packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -146,6 +183,13 @@ export class ServerManager {
       return { pid, started: false };
     }
 
+    const npmResult = await npmCheck('@neo-one/server', this.checkPath, CHECK_TIMEOUT_MS, CHECK_DELAY_SEC);
```

Comment:
Should still check even if the server is already alive.

Reply:
👍